go/interpreter/lfvm/instructions.go

```diff
@@ -22,16 +22,15 @@ func opStop() status {
 	return statusStopped
 }
 
-func opRevert(c *context) status {
-	c.resultOffset = *c.stack.pop()
-	c.resultSize = *c.stack.pop()
-	return statusReverted
-}
-
-func opReturn(c *context) status {
-	c.resultOffset = *c.stack.pop()
-	c.resultSize = *c.stack.pop()
-	return statusReturned
+func opReturningOp(c *context, status status) (status, error) {
```

Comment:
"op" is present twice in this function name .. how about
```suggestion
func opEndWithResult(c *context, status status) (status, error) {
```